firefox-ios/Client/metrics.yaml

```diff
@@ -656,6 +656,67 @@ downloads:
       - fx-ios-data-stewards@mozilla.com
     expires: "2025-01-01"
 
+# Microsurvey
+microsurvey:
+  survey_message_shown:
```

Comment:
As we have the category `microsurvey`, I think we may remove `survey_message`, what do you think?

Reply:
works for me since we have the description

Reply:
👍🏽